packages/cli/src/validators/createTableDefOverriderList.ts

```diff
@@ -0,0 +1,36 @@
+import { ColumnDefProperties } from '@synthql/queries';
+import { SchemaDefOverrides } from '../types/CliConfig';
+
+interface TableDefOverrider {
+    isValidQualifiedTableName: (qualifiedTableName: string) => boolean;
```

Comment:
"valid" is the wrong adjective here. I would call this `test`, `predicate`, `isApplicable`.